cdap-app-fabric/src/main/java/io/cdap/cdap/internal/app/runtime/monitor/RuntimeClientService.java

```diff
@@ -95,7 +98,13 @@ public class RuntimeClientService extends AbstractRetryableScheduledService {
     this.fetchLimit = cConf.getInt(Constants.RuntimeMonitor.BATCH_SIZE);
     this.programFinishTime = new AtomicLong(-1L);
     this.topicRelayers = RuntimeMonitors.createTopicConfigs(cConf).entrySet().stream()
```

Comment:
Seems better to use a List everywhere instead of a LinkedHashMap so that it's more clear that the order is important. It looks like we don't even lookup by the key in this map anyway.

Reply:
Done